### ISSUE TITLE
Exception when working directory is not exisiting anymore

### ISSUE THREAD

**Issue by wvogl** [spyder-ide/spyder#5504]
## Description
An FileNotFound exception is raised when the working folder set in the IDE is not existing anymore (ie. folder has been deleted or moved) and you open a file. 
It also happens when selecting a working directory from the drop-down list which is not existing anymore. 

**What steps will reproduce the problem?**

1. Select a working folder in the IDE
2. Delete or move that folder
3. Press Open File icon

1. Delete or move a folder that is in the drop-down list of working folders
2. Select that folder in the drop-down list

**What is the expected output? What do you see instead?**
No exception, maybe switch to a default folder. 

**Please provide any additional information below**

  File "/home/wvogl/virtenv_3.6/lib/python3.6/site-packages/spyder/plugins/workingdirectory.py", line 286, in chdir
    os.chdir(directory)
FileNotFoundError: [Errno 2] No such file or directory: '/home/wvogl/workingFolder'

## Version and main components

* Spyder Version: 3.2.3 
* Python Version: 3.6.3
* Qt Versions: 5.9.1, PyQt5 5.9 on Linux

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.20.3 (OK)
numpy >=1.7      :  1.13.3 (OK)
sphinx >=0.6.6   :  1.6.4 (OK)
rope >=0.9.4     :  0.10.7 (OK)
jedi >=0.9.0     :  0.11.0 (OK)
psutil >=0.3     :  5.3.1 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
sympy >=0.7.3    :  1.1.1 (OK)
cython >=0.21    :  0.27.1 (OK)
qtconsole >=4.2.0:  4.3.1 (OK)
IPython >=4.0    :  6.2.1 (OK)
pylint >=0.25    :  1.7.4 (OK)

```

**Comment by wvogl** [comment]
Seems to be fixed in 3.2.4.

**Comment by ccordoba12** [comment]
Yep, we fixed this error in 3.2.4.
